tests/std/tests/LWG3234_math_special_overloads/test.cpp

```diff
@@ -0,0 +1,573 @@
+    static_assert(is_same_v<decltype(sph_bessel(0u, 0ull)), double>);
+    static_assert(is_same_v<decltype(sph_bessel(0u, 0.0)), double>);
+    static_assert(is_same_v<decltype(sph_bessel(0u, 0.0l)), long double>);
+    static_assert(is_same_v<decltype(sph_besself(0u, 0)), float>);
+    static_assert(is_same_v<decltype(sph_bessell(0u, 0)), long double>);
+    static_assert(is_same_v<decltype(sph_bessel(0u, Ambiguous{})), Ambiguous>);
+    assert(expect_epsilons(sph_bessel(1, 2), sin(2) / (static_cast<double>(2) * 2) - cos(2) / 2, 2));
+}
+
+void test_sph_legendre() {
+    static_assert(is_same_v<decltype(sph_legendre(0u, 0u, 0.0f)), float>);
+    static_assert(is_same_v<decltype(sph_legendre(0u, 0u, false)), double>);
+    static_assert(is_same_v<decltype(sph_legendre(0u, 0u, static_cast<unsigned short>(0))), double>);
+    static_assert(is_same_v<decltype(sph_legendre(0u, 0u, 0)), double>);
+    static_assert(is_same_v<decltype(sph_legendre(0u, 0u, 0u)), double>);
+    static_assert(is_same_v<decltype(sph_legendre(0u, 0u, 0l)), double>);
+    static_assert(is_same_v<decltype(sph_legendre(0u, 0u, 0ul)), double>);
+    static_assert(is_same_v<decltype(sph_legendre(0u, 0u, 0ll)), double>);
+    static_assert(is_same_v<decltype(sph_legendre(0u, 0u, 0ull)), double>);
+    static_assert(is_same_v<decltype(sph_legendre(0u, 0u, 0.0)), double>);
+    static_assert(is_same_v<decltype(sph_legendre(0u, 0u, 0.0l)), long double>);
+    static_assert(is_same_v<decltype(sph_legendref(0u, 0u, 0)), float>);
+    static_assert(is_same_v<decltype(sph_legendrel(0u, 0u, 0)), long double>);
+    static_assert(is_same_v<decltype(sph_legendre(0u, 0u, Ambiguous{})), Ambiguous>);
+
+    const double pi = acos(-1.0);
+    assert(expect_epsilons(sph_legendre(3, 0, 1), 0.25 * sqrt(7 / pi) * (5 * pow(cos(1), 3) - 3 * cos(1)), 2));
+}
+
+void test_sph_neumann() {
+    static_assert(is_same_v<decltype(sph_neumann(0u, 0.0f)), float>);
+    static_assert(is_same_v<decltype(sph_neumann(0u, false)), double>);
+    static_assert(is_same_v<decltype(sph_neumann(0u, static_cast<unsigned short>(0))), double>);
+    static_assert(is_same_v<decltype(sph_neumann(0u, 0)), double>);
+    static_assert(is_same_v<decltype(sph_neumann(0u, 0u)), double>);
+    static_assert(is_same_v<decltype(sph_neumann(0u, 0l)), double>);
+    static_assert(is_same_v<decltype(sph_neumann(0u, 0ul)), double>);
+    static_assert(is_same_v<decltype(sph_neumann(0u, 0ll)), double>);
+    static_assert(is_same_v<decltype(sph_neumann(0u, 0ull)), double>);
+    static_assert(is_same_v<decltype(sph_neumann(0u, 0.0)), double>);
+    static_assert(is_same_v<decltype(sph_neumann(0u, 0.0l)), long double>);
+    static_assert(is_same_v<decltype(sph_neumannf(0u, 0)), float>);
+    static_assert(is_same_v<decltype(sph_neumannl(0u, 0)), long double>);
+    static_assert(is_same_v<decltype(sph_neumann(0u, Ambiguous{})), Ambiguous>);
+    assert(expect_epsilons(sph_neumann(1, 1), -std::cos(1) - sin(1), 2));

```

Comment:
We're `using namespace std`:
```suggestion
    assert(expect_epsilons(sph_neumann(1, 1), -cos(1) - sin(1), 2));
```